<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_parcer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>441</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_cnt_facets">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>570</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Поверхностей: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cnt_vertex">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>570</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во вершин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_file">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип линий:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_line_mode_1">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>32</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Сплошная</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_line_mode_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Пунктирная</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Толщина линий</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_line_width">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_line_mode_0">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Откл.</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>311</y>
      <width>121</width>
      <height>168</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип точек:</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>118</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер точек</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_vertex_size">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>140</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_vertex_mode_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_vertex_mode_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Круг</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_vertex_mode_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>88</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Откл.</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет:</string>
    </property>
    <widget class="QSlider" name="slider_color_g">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_color_b">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_color_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_color_vertex">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Точек</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_color_background">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Фона</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_color_line">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Линий</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Перемещение:</string>
    </property>
    <widget class="QLabel" name="label_move_step">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_down">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Вниз</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_far">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Дальше</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_near">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ближе</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBox_move">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.300000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_right">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_left">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_up">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Вверх</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_left_x">
     <property name="geometry">
      <rect>
       <x>372</x>
       <y>50</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_right_y">
     <property name="geometry">
      <rect>
       <x>402</x>
       <y>80</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_left_y">
     <property name="geometry">
      <rect>
       <x>372</x>
       <y>80</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotate">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Поворот:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotate_z">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>110</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBox_zoom">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotate_x">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>50</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_right_z">
     <property name="geometry">
      <rect>
       <x>402</x>
       <y>110</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zoom">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштаб:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_right_x">
     <property name="geometry">
      <rect>
       <x>402</x>
       <y>50</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate_left_z">
     <property name="geometry">
      <rect>
       <x>372</x>
       <y>110</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_zoom_up">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotate_y">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>80</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_zoom_down">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpinBox_rotate">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>720</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_axis">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Оси координат</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_settings">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранять настройки</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип проекции:</string>
    </property>
    <widget class="QRadioButton" name="radioButton_projection_mode_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>60</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Параллельная</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_projection_mode_1">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>32</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Центральная</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>731</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="openglLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Запись:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_jpeg">
        <property name="text">
         <string>jpeg</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_bmp">
        <property name="text">
         <string>bmp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_screenshot">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Screenshot</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gif_progress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>GIF</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_loading_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
